input/fsh/valuesets/au-erequesting-coverage-self-pay.fsh

```diff
@@ -0,0 +1,16 @@
+ValueSet: AUeRequestingCoverageSelfPay
+Id: au-erequesting-coverage-self-pay
+Title: "AU eRequesting Coverage Type and Self-Pay"
```

Comment:
What are the conventions for naming value sets that include a subset of codes from an existing FHIR value set? 
This value set appears to be using different conventions for name, title, and URL compared to the AU eRequesting Task Status value set. Can you provide reference to documented conventions, or draft conventions so we can review against those?

Reply:
Conventions for AU Base are in the wiki [here](https://github.com/hl7au/au-fhir-base/wiki).
I'll start a wiki for this IG. Internal ValueSet naming will align to <Realm> <IG name> <ValueSetName>. If it extends an international valueset then it will be suffixed with -extended.

Reply:
See also **ValueSet conventions** embedded in Dusi's doc...
https://confluence.hl7.org/display/HAFWG/HL7+AU+artefact+identification%2C+naming%2C+versioning%2C+and+publication+policies . If you wish to change these it's there for editing.

Reply:
Thanks Michael. I'm still a bit unclear about the difference between what's documented and what's applied in the value set. If I understand the conventions correctly, the eRequesting value set should have these properties:

> id=au-erequesting-coverage-type
> URL=http://hl7.org.au/fhir/ereq/ValueSet/au-erequesting-coverage-type
> name=AUeRequestingCoverageTypeAndSelf-PayCodes
> title=AU eRequesting Coverage Type and Self-Pay Codes

Is this the intent of the conventions? 

I also think the existing url "http://hl7.org.au/fhir/ereq/ValueSet/au-erequesting-coverage-self-pay" is to narrow as the the value set does not include selfpay concepts only, and that it should use the code system's representation 'selfpay' with no hyphen.

Reply:
Agree - for the sake of consistency we should keep artefact names the same as the HL7 International standard even if they use redundant words like "Codes" in the name of the ValueSet.